.buildkite/pipeline.yml

```diff
@@ -99,7 +99,33 @@ steps:
       CMAKE_FLAGS: "-DCMAKE_TOOLCHAIN_FILE=cmake/darwin-x86_64.cmake"
       RUN_TESTS: "false"
 
-  - label: "Java Integration Tests"
+  - label: "Build & test :cpp: for Windows_x86_64-RelWithDebInfo :windows:"
+    timeout_in_minutes: 180
+    agents:
+      provider: gcp
+      machineType: "c2-standard-16"
+      minCpuPlatform: "Intel Cascade Lake"
+      image: family/ml-cpp-1-windows-2016
+    commands:
+      - 'echo "checkout succeeded"'
+      - "& Get-ComputerInfo"
+      - "& .buildkite\\scripts\\steps\\build_and_test.ps1"
+    depends_on: "check_style"
+    key: "build_test_Windows-x86_64-RelWithDebInfo"
+    artifact_paths:
+      - "lib/*/unittest/boost_test_results.junit"
+      - "bin/*/unittest/boost_test_results.junit"
```

Comment:
Is there a good reason not to use `"*/*/unittest/boost_test_results.junit"` here? Then it would be totally clear that the idea is to preserve all the same files that are being interpreted as unit test results.

Same for the other platforms (unless it's impossible to use a double wildcard for some reason).

Reply:
Worth a try..